tabs-tabs-tabs/tabs.js

```diff
@@ -122,5 +122,12 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-highlight") { // highlights current tab and next tab
+    callOnActiveTab((tab, tabs) => {
+      next = (tab.index+1) % tabs.length;
```

Comment:
This isn't the next tab. If tab.index + 1 is equal to tabs.length, its 0, which is likely the first tab. I can see the need to pick another tab though, but the comment on the line above is inaccurate and could maybe be improved?